tests/unit/specs/components/governance/PageProposal.spec.js

```diff
@@ -71,7 +71,7 @@ describe(`PageProposal`, () => {
     expect(wrapper.element).toMatchSnapshot()
   })
 
-  it(`shows an error if the proposal couldn't be found`, () => {
+   it(`shows an error if the proposal couldn't be found`, () => {
```

Comment:
Maybe handle a test for the unknown proposer?

Reply:
Done!